libraries/AP_Baro/AP_Baro.h

```diff
@@ -177,6 +180,8 @@ class AP_Baro
         float altitude;                 // calculated altitude
         AP_Float ground_pressure;
         float p_correction;
+        uint8_t can_manager;
+        uint8_t uavcan_node;
```

Comment:
Yes, this is needed for hot-plug.
There can be several baros on several separate AP_UAVCAN drivers. So the only place where they come together is AP_Baro instance and so it needs to have a list of all them to differentiate registration.